src/Smart.Design.Console.Test/Smart.Design.Console.Test.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>false</IsPackable>
+
+        <RootNamespace>Opportunity.DesignSystem.Console.Test</RootNamespace>
```

Comment:
The `RootNamespace` and the `csproj` name are different but this needs to be consistent.

Reply:
Good catch, will do